src/commands/transaction/cosign.ts

```diff
@@ -107,8 +107,11 @@ export default class extends ProfileCommand {
             });
     }
 
-    private getGraphAccounts(profile: Profile): Observable<PublicAccount[]> {
-        return new MultisigHttp(profile.url).getMultisigAccountGraphInfo(profile.account.address)
+    private getGraphAccounts(
```

Comment:
We can refactor this function to not have to pass ``publicAccount``, by returning ``Observable<Address[]>``